orttraining/orttraining/training_api/checkpoint.cc

```diff
@@ -551,9 +552,18 @@ Status OrtLoadInternal(const PathString& checkpoint_path,
     LoadTensorProtoFromFile(tensor_file_full_path, tensor_protos, "[params]");
 
     for (const auto& tensor_proto : tensor_protos) {
-      param_tensor_protos.push_back(tensor_proto);
+      param_tensor_protos.insert(std::make_pair(tensor_proto.name(), tensor_proto));
     }
   }
+
+  // Load imported initializers into the Model
+  for (auto& init : *(model_proto.mutable_graph()->mutable_initializer())) {
+    if (!init.has_name()) ORT_THROW("Initializer name is missing.");
```

Comment:
Please describe the error in detail. Just initializer name missing is not enough to understand the exact issue.
Same comment for line 562 Invalid initializer name is not clear either.